bedrock/newsletter/templates/newsletter/country.html

```diff
@@ -1,39 +1,41 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. #}
+
 {% extends 'newsletter/base.html' %}
 
 {% block gtm_page_id %}data-gtm-page-id="/newsletter/country/"{% endblock %}
 
 {# "noindex" pages should not have the canonical or hreflang tags: bug 1442331 #}
 {% block canonical_urls %}<meta name="robots" content="noindex,follow">{% endblock %}
 
-{% block body_id %}newsletter-country{% endblock body_id %}
-
 {% block page_title %}Newsletter Country or Region{% endblock page_title %}
 
 {% block content %}
-  {% if switch('newsletter-maintenance-mode') %}
-    <div id="main-feature">
-      <p class="billboard">
-        This page is in maintenance mode and is temporarily unavailable.<br>
-        To update your email preferences, please check back in a little while. Thanks!
-      </p>
-    </div>
-  {% else %}
-    <div id="main-feature">
-      <h1>Thanks for updating your info with us!</h1>
-      <p>
-        Simply select your country or region from the list below and hit "submit."<br>
-        We've pre-filled what we think is correct, but make sure to check that it's right
-        before clicking the button. Thanks!
-      </p>
+<main role="main">
+  <div class="mzp-l-content mzp-t-narrow">
+    {% if switch('newsletter-maintenance-mode') %}
+    <div class="mzp-c-emphasis-box">
+      <p>This page is in maintenance mode and is temporarily unavailable.</p>
+      <p>To update your email preferences, please check back in a little while. Thanks!</p>
     </div>
+    {% else %}
+    <h1>Thanks for updating your info with us!</h1>
+    <p>
+      Simply select your country or region from the list below and hit "submit."<br>
```

Comment:
The `<br>` here creates a somewhat odd linebreak:

<img src="https://user-images.githubusercontent.com/400117/71969014-caf8a080-31fd-11ea-9353-5fd159573c89.png" width="500">

Could we split these two sentances into separate paragraphs? 

The select input also breaks out of the page bounds at mobile screen sizes:

<img src="https://user-images.githubusercontent.com/400117/71969237-24f96600-31fe-11ea-8cf3-906ee89bb208.png" width="300">

URL: http://localhost:8000/en-US/newsletter/country/d78f021e-459d-4d07-8588-1f2bb5420f18/